lib/galaxy/model/scoped_session.py

```diff
@@ -0,0 +1,15 @@
+"""
+These classes are used for registering different scoped_session objects with
+the DI framework. This type distinction is necessary because we need to store
+scoped_session objects that produce sessions that may have different binds
+(i.e., if the tool_shed_install model uses a different database).
+"""
+from sqlalchemy.orm import scoped_session
+
+
+class gxy_scoped_session(scoped_session):
```

Comment:
Can you rename this to `galaxy_scoped_session` or just `scoped_session` and `tsi_scoped_session` to `install_model_scoped_session` ?

Reply:
Done! `galaxy_scoped_session` and `install_model_scoped_session`.